shared/images/iconfont/README.md

```diff
@@ -62,41 +62,21 @@ The naming convention of the SVG files is very important.
 `{counter}-kb-inconfont-{name-with-dashes}-{size}.svg`
 
 The counter is used to generate the unicode values for the characters in the
-iconfont. It is okay to have gaps in the counters.
+iconfont. It is okay to have gaps in the counters. It is NOT ok to have
+multiple icons with the same counter.
 
 For instructions on adding/modifying icons look at the instructions in this
 [zeplin sheet](https://zpl.io/29y4w5w).
 
 ### Common Errors
 
 1. Flow: `Cannot create Kb.Icon element because property 'iconfont-{name}' is missing in object [1] in property 'type'`
-    - This happens when an icon name reference via the `type` prop on a `Kb.Icon` component, but Flow cannot find the matching `type` in `icon.constants.js`
-    - Check the naming of the correct file in `images/iconfont/*.svg`
-    - Also ensure that the key in the object was not deleted on accident.
-
-2. Fontforge not being installed or unavailable via `PATH` in the shell.
```

Comment:
We don't need this common error anymore?

Reply:
The script should be telling people this now
https://github.com/keybase/client/blob/ec1eec2e6b482ba2c4ba437abaf7c70dc24c97e3/shared/desktop/yarn-helper/font.js#L82-L89

Reply:
Awesome. 👍